### DISCUSSION TITLE
Help with installation

### DISCUSSION THREAD

**Discussion by RasmusRaschke** [quantum-visualizations/qmsolve#11] [Q&A] [answered]
Hello,
I never installed a GitHub repo before as I always used packages which are installed by "pip install" or "conda install". Now I used a GitHub-Client to clone the repo into my conda environment. However, if I want to run one of the example codes in my IDE, I get the following error: 
Traceback (most recent call last):
  File "...", line 2, in <module>
    from qmsolve import Hamiltonian, SingleParticle, init_visualization
ImportError: cannot import name 'Hamiltonian' from 'qmsolve' (unknown location)
Any help would be appreciated as I really want to try out your package!

**Comment by rafael-fuente** [comment, marked as answer]
Hello,
We haven't uploaded it yet to PyPI, but we will soon.

The actual way to run your script or notebook is by placing them inside the folder you cloned, along with the rest of the examples (1D_harmonic_oscillator.py, 1D_interactive_fermions.py ... ). 
- The idea is that the [qmsolve ](https://github.com/quantum-visualizations/qmsolve/tree/main/qmsolve) folder must be inside the same folder of the script you want to run.

Also, If you are using a conda environment you can 'fake' the installation by manually placing the [qmsolve](https://github.com/quantum-visualizations/qmsolve/tree/main/qmsolve) folder in the following directory:

`$path to your conda installation$\Lib\site-packages`

If you are using Windows, that path may be `C:\Users\%USERNAME%`  or `C:\Program Files`
If you make the above, you will be able to run your script from any directory.
Note, however, that to uninstall it you must also delete the folder manually

**Reply by RasmusRaschke** [reply]
Thank you very much, I used the 'fake' installation and it works for some of the codes you provided. Now I get the error " AttributeError: 'Hamiltonian' object has no attribute 'Vmin' " whenever I use one of the codes that has the Hamiltonian function in it. Do you know if I did something wrong?
Anyway, I already begin to enjoy your package and look forward to all the features and improvements coming in the future.

**Reply by rafael-fuente** [reply]
My apologies, that was my fault. 
I am currently working on implementing momentum operators dependent potentials, so for example, charged particles in a magnetic field can be simulated. I renamed some hours ago `Vmin` variable to `Emin` and I forgot to also rename the variable in two lines.
I updated the repository and fixed the problem. You must replace the entire qmsolve folder.

Thank you!

**Reply by RasmusRaschke** [reply]
No problem, it works perfectly fine now.

**Reply by rafael-fuente** [reply]
Finally, the beta version of the package is pip installable:

`pip install qmsolve`

If you want to install the new version, remember to delete the 'fake' installation manually.

**Reply by RasmusRaschke** [reply]
Thanks, I will do it right away.
